api/Makefile

```diff
@@ -28,3 +28,4 @@ detect_missing_migrations:  ## Determine if any apps are missing generated migra
 
 migrate: ## Apply database migrations
 	python manage.py migrate --noinput
+	python manage.py collectstatic --no-input
```

Comment:
I would suggest adding a new alias for just collecting static files and another for both (e.g. `static`) since this doesn't really have to do with database migration.

Reply:
And then `make collectstatic` would also need to be added to `woeip/api/entrypoint.sh`.